framework_agreement/model/company.py

```diff
@@ -30,5 +30,5 @@ class res_Company(models.Model):
         'One agreement per product',
         help='If checked you can have only'
         ' one framework agreement '
-        ' per product at the same time'
+        ' per product at the same time.'
```

Comment:
@pedrobaeza good catch I always forgot to do it. It would be nice to have some automated tools/check to enforce this.